lib/ionic_notification/concerns/ionic_notificable.rb

```diff
@@ -2,17 +2,44 @@ module IonicNotification
   module Concerns
     module IonicNotificable
       extend ActiveSupport::Concern
-      include IonicNotification::Logger
 
       included do
         serialize :device_tokens, Array
 
         def notify(options = {})
-          return missing_device_tokens_logger unless respond_to?(:device_tokens)
-          return no_device_tokens_logger(self) unless device_tokens.count > 0
-          notification = IonicNotification::Notification.new(options.merge!(tokens: device_tokens))
+          logger = self.class.new_logger
+          return logger.missing_device_tokens unless respond_to?(:device_tokens)
+          tokens = device_tokens.uniq
+          return logger.no_device_tokens(self) unless tokens.count > 0
+          notification = IonicNotification::Notification.new(options.merge!(tokens: tokens))
           notification.send
         end
+
+        def notify_at(schedule, options = {})
+          options.merge!(scheduled: schedule.to_time.to_i)
```

Comment:
Do not use `to_time` on Date objects, because they know nothing about the time zone in use.